test/test_torch.py

```diff
@@ -15457,6 +15378,56 @@ def test_min_with_inf(self, device, dtype):
         self.assertTrue(torch.all(torch.min(a, dim=1)[0] == (-inf)).item())
         self.assertTrue(torch.min(a).item() == -inf)
 
+    def _test_minmax_helper(self, torchfn, reffn, device, dtype, skip_indices=False):
+        def create_input(shape, device, dtype):
+            if dtype.is_floating_point:
+                return torch.randn(*shape, device=device, dtype=dtype)
+            else:
+                low = 0 if dtype == torch.bool else -1000
+                high = 2 if dtype == torch.bool else 1000
+                return torch.randint(low, high, shape, device=device, dtype=dtype)
+        x = create_input((100, 100), device, dtype)
+        self.compare_with_numpy(torchfn, reffn, x)
+        # non contiguous
+        x = create_input((10, 10, 10), device, dtype)
+        x = x[:, 4]
+        self.compare_with_numpy(torchfn, reffn, x)
+        # indices
+        if not skip_indices:
```

Comment:
You could create a tensor and its transpose as inputs, then enumerate over the product of dim x input here for more thorough coverage of various dims and the dim x noncontiguous case.

Reply:
(If you really wanted you to could product in keepdims, too

Reply:
Pass on keepdims, done for transpose and dim.